mantle/kola/tests/rhcos/upgrade.go

```diff
@@ -358,8 +359,23 @@ func downloadLatestReleasedRHCOS(target string) (string, error) {
 		return
 	}(releaseIndex, unique)
 
+	// In cases where there is a blocked upgrade to a new Y-stream there can be
+	// two nodes that don't have an edge to upgrade to. This is generally the
+	// latest 4.Y-1.Z and the latest 4.Y.Z. Choose the latest 4.Y.Z
+	latest := difference[0]
+	if len(difference) == 2 {
```

Comment:
Would it make sense to make this more generic for `> 1`? I.e. loop over every item in `difference` and pick the latest one?

Reply:
Thanks! updated